### DISCUSSION TITLE
Wasm support.

### DISCUSSION THREAD

**Discussion by opeolluwa** [tursodatabase/turso#515] [Q&A] [answered]
I'm building a Tauri/wasm application, targeting wasm32-unknown-unkown environment.

Which wasm environment is supported. And is there Rust-wasm bindings for use in Rust wasm frameworks

**Comment by penberg** [comment, marked as answer]
Hey @opeolluwa! We currently only build for the "nodejs" target with "wasm-pack". Not sure what else is needed for your use case. You can find the bindings here: https://github.com/tursodatabase/limbo/tree/main/bindings/wasm

**Reply by opeolluwa** [reply]
Thanks!

**Comment by jeet-dhandha** [comment]
@penberg Does it work for front-end frameworks like vite, react, nextjs, etc  🤔 . 
I m planning to build a front-end first application. So its mostly storing millions of rows, making aggregations on it, and many similar features.

**Comment by forfun3d** [comment]
Would be nice to use it in rust by giving it access to sync handles (db/wal) in rust running in wasm in a worker.
Maybe its possible already ?

**Reply by opeolluwa** [reply]
absolutely
